code/game/machinery/computer/computer.dm

```diff
@@ -19,6 +19,12 @@
 	overlay_layer = layer
 	..()
 
+/obj/machinery/computer/Destroy()
```

Comment:
It's no big deal; I overlooked it too; just pointing it out so that when you continue work on this, it won't continue to be a problem for ya.